tests/sentry/api/endpoints/test_organization_member_index.py

```diff
@@ -474,41 +474,41 @@ def test_manager_invites(self):
         )
         self.login_as(user=manager_user)
 
-        data = {"email": "eric@localhost", "role": "owner", "teams": [self.team.slug]}
+        data = {"email": "eric1@localhost", "role": "owner", "teams": [self.team.slug]}
         self.get_error_response(self.organization.slug, **data, status_code=400)
 
-        data = {"email": "eric@localhost", "role": "manager", "teams": [self.team.slug]}
-        self.get_success_response(self.organization.slug, **data)
+        data = {"email": "eric2@localhost", "role": "manager", "teams": [self.team.slug]}
+        self.get_success_response(self.organization.slug, **data, status_code=201)
 
-        data = {"email": "eric@localhost", "role": "member", "teams": [self.team.slug]}
-        self.get_error_response(self.organization.slug, **data, status_code=400)
+        data = {"email": "eric3@localhost", "role": "member", "teams": [self.team.slug]}
+        self.get_success_response(self.organization.slug, **data, status_code=201)
```

Comment:
This was always a successful invite, but it failed because the email was identical.
I changed the email for all the tests so each invite w/ a diff role has a diff email